Platon/backend/app/search_engine/forms.py

```diff
@@ -70,4 +71,23 @@ class UpcomingEventsSearchForm(Form):
 upcoming_events_search_parser.add_argument('sorting_criteria',type=int,help="None => Semantic Rating // 0 => Alphabetical Order(A=>Z) // 1 => Date Order",location='args')
 upcoming_events_search_parser.add_argument('page',type=int,help="Page ID(0-Indexed)",location='args')
 upcoming_events_search_parser.add_argument('per_page',type=int,help="Number of Records in a Page",location='args')
-upcoming_events_search_parser.add_argument('auth_token', type=str,help="Authantication Token(If registered)",location='headers')
\ No newline at end of file
+upcoming_events_search_parser.add_argument('auth_token', type=str,help="Authantication Token(If registered)",location='headers')
+
+def validate_json(form, field):
+	try:
+		json.loads(field.data)
+	except ValueError as e:
+		raise ValidationError("This is not a valid JSON string.")
+
+class TagSearchForm(Form):
+    search_type = IntegerField("search_type",validators=[validators.NumberRange(min=0, max=1),validators.InputRequired()])
+    skills = StringField("skills", validators=[validators.InputRequired(), validate_json])
```

Comment:
Can you give an example of "skills" field? I'm confused a bit.

Reply:
[“C++”,”Java”]

Reply:
[“C++”]